src/test/test-packages/08-create-differential-git/README.md

```diff
@@ -0,0 +1,3 @@
+# Git Data
+
+This test package simply iterates the version of the Git Data package so a differential package can be created (and tested).
```

Comment:
You know what you could do... Make the version a package template... (we could keep the package in testing but it would be nice to have one package definition)

Reply:
To get a bit of clarification; Adding a package template to the example would mean that people would have to provide the `--set` flag whenever they build that package. When you say "it would be nice to have one package definition" does that mean you're ok with that behavior for those two seemingly random example packages?